k8s/core/nodes.go

```diff
@@ -177,7 +177,7 @@ func (c *Client) SearchNodeByAddresses(addresses []string) (*corev1.Node, error)
 
 	// sweep #3 - locating based on labels
 	for _, node := range nodes.Items {
-		if hn, has := node.GetLabels()[corev1.LabelHostname]; has {
+		if hn, has := node.GetLabels()[labelHostname]; has {
```

Comment:
Why not use the label constant directly from kubernetes library?

Reply:
This label has been added to the api package v1.14.x. This introduces an issues for projects which depend on older k8s packages.